src/main/java/com/exasol/releaserobot/repository/maven/MavenPomParser.java

```diff
@@ -1,40 +1,37 @@
 package com.exasol.releaserobot.repository.maven;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Optional;
+import java.io.*;
+import java.util.*;
 
-import javax.xml.parsers.*;
-
-import org.apache.commons.io.IOUtils;
-import org.w3c.dom.*;
-import org.xml.sax.SAXException;
+import org.apache.maven.model.*;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
 
 import com.exasol.releaserobot.repository.GitRepositoryException;
 
 /**
  * Parser for {@link MavenPom}.
  */
 public class MavenPomParser {
```

Comment:
this class should implement an interface, mainly cause if at some point you want to change, ie, from apache to another library you don't have to change the clients of this class.

For the moment it's ok, but at some point in the future this should be changed.